examples/libraries/libcurl/download_image/README.md

```diff
@@ -0,0 +1,3 @@
+# Example using libcurl to download an image and generate some ASCII art reading it with stb
+
+This example is used in the Conan 2.X CLion plugin blogpost: https://blog.conan.io/
```

Comment:
```suggestion
This example is used in the Conan 2.X CLion plugin blogpost: https://blog.conan.io/introducing-new-conan-clion-plugin/
```

Reply:
Actual links are lik https://blog.conan.io/2023/07/20/introduction-to-game-dev-with-sdl2.html, including the date

Reply:
I used a permalink in the post so that we could release the plugin without having to know the real release date: https://github.com/conan-io/conan-io.github.io/pull/253/files#r1308339266